lms/djangoapps/discussion_api/tests/test_api.py

```diff
@@ -645,107 +645,67 @@ def test_basic_query_params(self):
 
     def test_thread_content(self):
         source_threads = [
-            {
-                "type": "thread",
+            make_minimal_cs_thread({
                 "id": "test_thread_id_0",
                 "course_id": unicode(self.course.id),
                 "commentable_id": "topic_x",
-                "group_id": None,
-                "user_id": str(self.author.id),
                 "username": self.author.username,
-                "anonymous": False,
-                "anonymous_to_peers": False,
-                "created_at": "2015-04-28T00:00:00Z",
-                "updated_at": "2015-04-28T11:11:11Z",
-                "thread_type": "discussion",
+                "user_id": str(self.author.id),
                 "title": "Test Title",
                 "body": "Test body",
-                "pinned": False,
-                "closed": False,
-                "abuse_flaggers": [],
                 "votes": {"up_count": 4},
                 "comments_count": 5,
                 "unread_comments_count": 3,
                 "endorsed": True,
                 "read": True,
-            },
-            {
-                "type": "thread",
+                "created_at": "2015-04-28T00:00:00Z",
```

Comment:
I'm not sure how we determined which of these fields needs to be overridden for this to be a useful test?